docs/my-website/docs/proxy/logging.md

```diff
@@ -41,7 +41,8 @@ litellm_settings:
 **Step 3**: Set required env variables for logging to langfuse
 ```shell
 export LANGFUSE_PUBLIC_KEY="pk_kk"
-export LANGFUSE_SECRET_KEY="sk_ss
+export LANGFUSE_SECRET_KEY="sk_ss"
+export LANGFUSE_HOST="https://xxx.langfuse.com"
```

Comment:
please mark this as optional in the comment

and specify what the default value is when it's not passed

Reply:
Got it!